src/components/Button/button.test.js

```diff
@@ -0,0 +1,40 @@
+import Button from '.';
+import { fireEvent, render, screen } from '../../utils/tests/renderWithRouter';
+
+const handleClick = (event) => {
+  event.target.textContent = 'Clicked';
+};
```

Comment:
```suggestion
```

This is a bit weird way to test if the button has been clicked. You can define a new constant like this:

`const onClick = jest.fn();`

And pass it as a component prop on the render function:

`render(<Button onClick={onClick}>Press ME</Button>);`

After that, you can test if the button has been clicked by doing the following after the FireEvent function:

`expect(onClick).toHaveBeenCalled();`